packages/wallet/backend/src/paymentPointer/service.ts

```diff
@@ -200,7 +201,23 @@ export class PaymentPointerService implements IPaymentPointerService {
       accountId,
       paymentPointerId
     )
-    await paymentPointer.$query().patch({ keyIds: null })
+
+    if (!paymentPointer.keyIds) {
+      return
+    }
+
+    const trx = await PaymentPointer.startTransaction()
+    try {
+      await Promise.all([
+        await paymentPointer.$query(trx).patch({ keyIds: null }),
+        await this.deps.rafikiClient.revokePaymenterPointerKey(
```

Comment:
what is paymenter? I've seen it before in the code, Should it be paymentPointer?